cmd/sovereignnode/incomingHeader/incomingEventsProcessor.go

```diff
@@ -85,26 +98,29 @@ func (iep *incomingEventsProcessor) processIncomingEvents(events []data.EventHan
 
 func (iep *incomingEventsProcessor) createSCRInfo(topics [][]byte, event data.EventHandler) (*scrInfo, error) {
 	// TODO: Check each param validity (e.g. check that topic[0] == valid address)
-	if len(topics) < minTopicsInTransferEvent || len(topics[1:])%numTransferTopics != 0 {
+	if len(topics) < minTopicsInTransferEvent || len(topics[2:])%numTransferTopics != 0 {
```

Comment:
could be extracted to a new func checkTopicsValidity(topics) if more checks have to be made in the future

Reply:
yes, we'll need do add more checks, as the `TODO` says